build-support/docker/Verify-Release-Fedora.dockerfile

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+FROM fedora:36
```

Comment:
Wondering if we should use `fedora:latest` here to avoid needing to remember to update

Reply:
resolved in 206971db6874891b7124e55617cf021c69910523